package.json

```diff
@@ -8,6 +8,9 @@
     "name": "jQuery Foundation and other contributors",
     "url": "https://github.com/jquery/jquery-color/blob/master/AUTHORS.txt"
   },
+  "scripts": {
+    "ci": "grunt"
```

Comment:
grunt default task is defined as:

```js
grunt.registerTask( "default", [ "jshint", "qunit", "build", "compare_size" ] );
```

maybe we could just call `grunt jshint qunit` here.